File: newFive/src/main/java/formation/projetParis/newFive/entities/Maillot.java
package formation.projetParis.newFive.entities;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;

import com.fasterxml.jackson.annotation.JsonView;

@Entity
@DiscriminatorValue("M")
public class Maillot extends Marchandise {

	@JsonView(JsonViews.Base.class)
    private String equipe;
	@JsonView(JsonViews.Base.class)
    private String taille;

    public Maillot() {
    }

	public Maillot(String equipe, String taille) {
		super();
		this.equipe = equipe;
		this.taille = taille;
	}

	public String getEquipe() {
		return equipe;
	}

	public void setEquipe(String equipe) {
		this.equipe = equipe;
	}

	public String getTaille() {
		return taille;
	}

	public void setTaille(String taille) {
		this.taille = taille;
	}
    
    

}

File: newFiveAngular/src/app/model/maillot.ts
import { Marchandise } from "./marchandise";

export class Maillot extends Marchandise{
    constructor(
      private _equipe?: string,
      private _taille?: string
    ) {
      super()
     
    }
    public get equipe(): string | undefined {
      return this._equipe;
    }
    public set equipe(value: string | undefined) {
      this._equipe = value;
    }
  
    public get taile(): string | undefined {
      return this._taille;
    }
    public set taille(value: string | undefined) {
      this._taille = value;
    }
  }